lib/charms/hydra/v0/oauth.py

```diff
@@ -168,21 +171,21 @@ class DataValidationError(RuntimeError):
     """Raised when data validation fails on relation data."""
 
```

Comment:
You'll need to bump the `LIBPATCH` version

Reply:
Why? This is the first version of the library

Reply:
My bad! Indeed